configure/CONFIG_SITE

```diff
@@ -108,6 +108,7 @@
 #  Definitions in configure/os/CONFIG_SITE.<host>.Common
 #  may override this setting.
 CROSS_COMPILER_TARGET_ARCHS=
+CROSS_COMPILER_TARGET_ARCHS+=$(EPICS_HOST_ARCH)-debug
```

Comment:
If you put local config changes like this in a new file configure/CONFIG_SITE.local they will be read in but won't normally get committed to your git repo by mistake.